crates/relayer-types/src/applications/transfer/packet.rs

```diff
@@ -45,3 +47,60 @@ impl From<PacketData> for RawPacketData {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(try_from = "RawPacketDataV2", into = "RawPacketDataV2")]
+pub struct PacketDataV2 {
+    pub token: Vec<PrefixedCoin>,
```

Comment:
```suggestion
    pub tokens: Vec<PrefixedCoin>,
```

Reply:
done in fc41cdceb1b3993243cbf406faf415bc098f2599